src/routes/components/components.json

```
@@ -1,4 +1,77 @@
 [
+	{
+		"title": "sveltekit-adapter-auto",
+		"url": "https://github.com/sveltejs/kit/tree/master/packages/adapter-auto",
+		"description": "Automatically chooses adapter for your current environment if possible",
+		"npm": "https://www.npmjs.com/package/@sveltejs/adapter-auto",
+		"addedOn": "2022-06-09",
+		"tags": ["integrations"],
+		"category": "Sveltekit Adapters",
+		"stars": 8700
+	},
+	{
+		"title": "sveltekit-adapter-cloudflare-workers",
+		"url": "https://github.com/sveltejs/kit/tree/master/packages/adapter-cloudflare-workers",
+		"description": "SvelteKit adapter that creates a Cloudflare Workers site using a function for dynamic server rendering.",
+		"npm": "https://www.npmjs.com/package/@sveltejs/adapter-cloudflare-workers",
```

The `npm` node is only the NPM package name, not the URL

```suggestion
		"npm": "@sveltejs/adapter-cloudflare-workers",
```